### DISCUSSION TITLE
Thumbnail views instead of live video

### DISCUSSION THREAD

**Discussion by dnestico** [mrlt8/docker-wyze-bridge#308] [Q&A] [answered]
How can I make it that instead of loading the live video which seems to lag/buffer and also stop working a lot with the 4 cameras I have, I want it that instead, it shows thumbnails which are updated lets say every 5-15 seconds if possible (or on motion), then when the picture/thumbnail is clicked it will load the live view. 

I think this might help with my problem of slow loading and also the live stream breaking a lot since it won't be showing live video anymore unless tapped.

**Comment by mrlt8** [comment]
Are you on v1.2.0? There were some changes that should improve the streams a little.

Streaming on demand is something that's been on my todo list for a while now.

As for motion triggers, I've been searching for a way to do it over the tutk protocol, but have yet to figure out a way to access the camera's motion detection.

**Reply by dnestico** [reply]
Yes I'm on 1.2.0... I changed it to auto because I have a it on a wall mounted dashboard that wakes up via motion and the cameras take some time to load and last night they just went completely offline on HA, not in the wyze app though, I changed some of the settings in the config like I turned on "WEBRTC" and "FRESH_DATA" and then restarted so still testing to see if that helps. I'd say the least unstable camera is the doorbell as that stream is a hit or miss sometimes, would love for that one to be able to be triggered via the button btw that would be a great front door automation.

By the way if I force either LAN or P2P do you think one of those might make the streams load faster and be more stable?

And yea motion triggers would be amazing! I'm gonna test using the "RTSP:15" (is it possible to make that 15 seconds down to 10 maybe?)  images to see if I can put that in a picture card and then have it when you tap it the video loads to see if that works later for the on demand thing btw.

**Comment by dnestico** [comment, marked as answer]
So thanks to your plugin which got the camera feed in home assistant and then using the "proxy camera" config thing I found...

I've now figured how to get snapshots on my home assistant notifications easily!


![Screenshot_20220305-065006.png](https://user-images.githubusercontent.com/3034555/156881805-113c6319-740f-4502-a8dc-8cb53d2c0635.png)
